Assets/_BForBoss/_Character/Scripts/PlayerWallRunBehaviour.cs

```diff
@@ -245,10 +245,16 @@ public float CalculateWallSideRelativeToPlayer()
         
         private bool ShouldStartWallRun(out RaycastHit hit)
         {
-            var shouldStartWallRun =
-                ProcessRaycasts(_startWallRunDirections, out hit, ChildTransform, _parkourWallMask, _wallMaxDistance)
-                && IsClearOfGround()
-                && hit.collider != _lastWall;
+            var isRayCastingLastWallFromCenterBodyPosition = ProcessRaycasts(_startWallRunDirections, out hit,
```

Comment:
up to you, but you can make a helper function to shorten the variable declaration which I think could be better for readability

```
private bool isRaycastingLastWall(Transform pivot)
{
   return ProcessRaycasts(_startWallRunDirections, out hit, pivot, _parkourWallMask, _wallMaxDistance) && hit.collider != lastWall;
}
```
then declare the variables as 
`var isRaycastingLastWallFromCenterBodyPosition = isRaycastingLastWall(ChildTransform);`
`var isRaycastingLastWallFromEyePosition = isRaycastingLastWall(ChildTransform);`

Reply:
Normally writing a function just for that one line of code is a code smell. (case by case)

as long you already understand clearly what i'm trying to express there, I think it's fine. 
But if you still think it's better for readability then i can extract it.